LICENSE

```diff
@@ -0,0 +1,633 @@
+Copyright (C) 2019  Dirk Kelly breadtube@dirkkelly.com
```

Comment:
I think we should block the license until we just have the domain set up as an email.

`Copyright (C) 2019 BreadTube.tv license@breadtube.tv`